### PR TITLE
move routes to one place

### PR DESCRIPTION
**Pull request by a-popova**
1. Place all routes in one place - app.js
2. Separate Dashboard as column-left and reflect it in Profile, Reservation, History components

Question: is it ok to organise logic wih routes /profile, /reservation, /history like this or should I make changes? 
Previously, when router was nested inside Dashboard component, the page wouldn't reload each time, only column-right needed to reload. Now for each new route page must reload. What do you think about it?

### CODE REVIEW

**Comment by a-popova** [comment]
1. change paths in dashboard
2. changed signup logic

File: client/src/components/signup/signup.js [hunk 1]
```diff
@@ -18,12 +19,12 @@ const Signup = (props) => {
...
+  let history = useHistory();
 
-  const onSubmit = (userData) => {
+  const onSubmit = async (userData) => {
     if (userData) {
-      props.register(userData);
-      if (created) {
+      const isSuccessful = await props.register(userData);
+      if (isSuccessful) {
         history.push('/login');
```
Comment (DashaKavalenka) [inline client/src/components/signup/signup.js:28]: if not successful  catch error

File: client/src/services/userService.js [hunk 2]
```diff
@@ -25,13 +25,13 @@ function logout() {
...
-      'http://localhost:3000/api/users',
+      '/api/users',
       { userData },
       { header },
     );
-    return res;
+    return JSON.stringify(res.data);
   } catch (error) {
-    console.error(error);
+    return Promise.reject(error);
```
Comment (DashaKavalenka) [inline client/src/services/userService.js:34]: Add [react-toastify](https://www.npmjs.com/package/react-toastify) and show error for user. Also 

```suggestion
    return throw error;
```
or with toast

```suggestion
  const { message } = error.response.data

        toast.error(message, {
          autoClose: 2000,
        })
      }
```

Comment (DashaKavalenka) [reply client/src/services/userService.js:34]: String 15 unnecessary return

**Review by DashaKavalenka — COMMENTED** [review]
Move and  rename client/src/store/actions.js ->client/src/store/actions/authActions

**Review by DashaKavalenka — APPROVED** [review]

**Pull request merged**